users/permissions.py

```diff
@@ -17,3 +18,38 @@ def has_permission(self, request, view):
     def has_object_permission(self, request, view, obj):
         """Object-level permissions for user. This determines whether the request can access a user instance or not."""
         return request.user.is_superuser or request.user == obj
+
+
+class OrganizationUserPermission(permissions.BasePermission):
+    """
+    Permission check for organization-user mapping.
+    """
+
+    def has_permission(self, request, view):
+        """View-level permissions for organization-user. This determines whether the request can access organization-user instances or not."""
+        if request.user.is_superuser:
+            return True
+
+        if view.action == "create":
+            # only organization admins can add a user to their organization
+            user_is_organization_admin = OrganizationUser.objects.filter(
+                organization=request.data["organization"],
+                user=request.user.id,
+                role__name="org-admin",
```

Comment:
there's a `super-admin` role too, right?

Reply:
cannot update things if I am `super-admin` of an org right now.

Reply:
I think there is some confusion here also.

So what's the difference between `super-admin` and `org-admin` of an organization?

Is `super-admin` supposed to be same as `superuser`?